app-frontend/src/app/pages/projects/edit/annotate/import/import.scss

```diff
@@ -0,0 +1,21 @@
+@import "../../../../../../assets/styles/sass/settings/_colors.scss";
+
+#btn-upload {
```

Comment:
We try to avoid using id selectors wherever possible. I think this could be done using classes instead.